### ISSUE TITLE
WVDLogix AutoScaling Tool 0.2

### ISSUE THREAD

**Issue by bderry71** [wvdlogix/WVDLOGIX-Scripts#2]
New error with GUI App. Looks like you forgot to prompt for subscription like the old script did. A user might be a member of more than one Subscription.

Get-AzResourceGroup : No tenant found in the context.  Please ensure that the credentials you provided are authorized
to access an Azure subscription, then run Connect-AzAccount to login.
At C:\Scripts\wvd\WVDLogixAutoScale\WVDLogixAutoScale.ps1:65 char:25
+     $AutomationRGName = Get-AzResourceGroup | Out-GridView -PassThru  ...
+                         ~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzResourceGroup], ArgumentException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.GetAzureResourceGroupCmd
   let

Get-AzLocation : No subscriptions can be found under current directory. Please login again using Connect-AzAccount.
At C:\Scripts\wvd\WVDLogixAutoScale\WVDLogixAutoScale.ps1:67 char:27
+     $AutomationLocation = Get-AzLocation | Select DisplayName,Locatio ...
+                           ~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Get-AzLocation], PSInvalidOperationException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.ResourceManager.Cmdlets.Implementation.GetAzureLocationCmdlet

Unblock-File : Cannot find path
'C:\Users\Benjamin.Derry\Documents\WindowsPowerShell\Modules\AzFilesHybrid\0.1.2.0\AzFilesHybrid.psm1' because it does
not exist.
At C:\Scripts\wvd\WVDLogixAutoScale\WVDLogixAutoScale.ps1:76 char:5
+     Unblock-File -Path C:\Users\$LoggedUser\Documents\WindowsPowerShe ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Users\Benjam...ilesHybrid.psm1:String) [Unblock-File], ItemNotFoundE
   xception
    + FullyQualifiedErrorId : FileNotFound,Microsoft.PowerShell.Commands.UnblockFileCommand

Get-RandomString : The term 'Get-RandomString' is not recognized as the name of a cmdlet, function, script file, or
operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try
again.
At C:\Scripts\wvd\WVDLogixAutoScale\WVDLogixAutoScale.ps1:77 char:18
+     $SSCertPWD = Get-RandomString -StringLength 8 -AlphanumericOnly $ ...
+                  ~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (Get-RandomString:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException

.\New-RunAsAccount.ps1 : The script 'New-RunAsAccount.ps1' cannot be run because it contains a "#requires" statement
for running as Administrator. The current Windows PowerShell session is not running as Administrator. Start Windows
PowerShell by  using the Run as Administrator option, and then try running the script again.
At C:\Scripts\wvd\WVDLogixAutoScale\WVDLogixAutoScale.ps1:111 char:5
+     .\New-RunAsAccount.ps1 -ResourceGroup $AutomationRGName.ResourceG ...
+     ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : PermissionDenied: (New-RunAsAccount.ps1:String) [], ScriptRequiresException
    + FullyQualifiedErrorId : ScriptRequiresElevation

Get-Content : Cannot find path 'C:\Scripts\WVD\WVDLogixAutoScale\Scripts\WVDLogixAutomation.log' because it does not
exist.
At C:\Scripts\wvd\WVDLogixAutoScale\WVDLogixAutoScale.ps1:115 char:16
+ ...  $WebHook = Get-Content $WorkingFolderPath"\Scripts\WVDLogixAutomatio ...
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ObjectNotFound: (C:\Scripts\WVD\...xAutomation.log:String) [Get-Content], ItemNotFoundEx
   ception
    + FullyQualifiedErrorId : PathNotFound,Microsoft.PowerShell.Commands.GetContentCommand

Exception calling "ShowDialog" with "0" argument(s): "Cannot bind argument to parameter 'SubscriptionId' because it is
null."
At C:\Scripts\wvd\WVDLogixAutoScale\WVDLogixAutoScale.ps1:205 char:1
+ $wpf.WVDLogixAutoScale1.ShowDialog()
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ParameterBindingValidationException

**Comment by bderry71** [comment]
Another issue is you are not prompting for Automation Group or AADTenantID or Location. Also I now see that we have another bug where if I create a new account in Microsoft 365 and attach it to only one Subscription. Then login with it on WVDLogix Autoscale Tool the tool reports that it is still logged in as my global admin which has multiple subscriptions associated with it. I'm not sure why me logging into the Microsoft login prompt is not being reflected in the session report in the PowerShell window or your GUI?
